extensions/community_extensions/skadnetwork.md

```diff
@@ -848,28 +852,51 @@ Used for intermediary SSP to SSP/DSP to DSP connections where a full list SKAdNe
 }
 ```
 
+
 ## IABTL SKAdNetwork ID List Proposal
 
 “IABTL SKAdNetwork ID List” is a common list of networks, DSPs, Advertisers and others who support Apple’s SKAdNetwork API. This list would serve a similar purpose that the TCF 2.0 [Global Vendor List][8] serves to identify common IDs in a compact range. It can be used in addition to the distributed lists supplied by SSPs and Networks or in place of those.
 
+In the IABTL list model, the total list of SKAdNetworks on a device should be the union of the IABTL list and raw SKAdNetworks IDs supplied in the bid stream minus those in the exclude list
+
 ### Participant responsibilities
 
 The responsibilities of each participant when using the SKAdNetwork specifications are as follows (in addition to the responsibilities at the beginning of this document).
 
+#### IABTL responsibilities are to:
+1. Organize SKAdNetwork IDs as well as define an automated self-serve registration process
+    - Not validate any input data
+2. Perform releases in batches at “x” cadence to ensure as many partners publishers have the most up-to-date lists
+    - List should be in both JSON and XML formats to allow publishers to build to the IABTL list as well as other lists
+3. Assign a permanent ID for each registred `SKAdNework ID`
+    - Each registrant may have more than `SKAdNetwork ID`. In this scenario, each `SKAdNetwork ID` will be assigned it's own unique IABTL ID
+4. Provide a tool for publishers to build their `info.plist` files and express IABTL signaling from various URLs and / or raw skadnetwork ID
+
+
+
 #### SSP/SDK responsibilities are to:
 
-4. (Optional) Assist intermediary buyers with accessing the range of IABTL supported SKAdNetwork IDs on the bid request, provided in a compact range format `skadnrng` where the full list of support SKAdNetwork IDs for that list can be retrieved
+1. (Optional) Assist intermediary buyers with accessing the range of IABTL supported SKAdNetwork IDs on the bid request, provided in a compact range format `skadnrng` where the full list of support SKAdNetwork IDs for that list can be retrieved
```

Comment:
is this still going to be relevant?

Reply:
Agreement on wg call that we should remove.

Reply:
Removed